src/components/wallet/components/GenerateWallet.js

```diff
@@ -1,83 +1,92 @@
 import React from 'react';
-import { connect } from 'dva';
-import { Link } from 'dva/router';
-import { Button,Form,Radio,Input,Tabs,Card,Badge,Icon,Modal,Progress } from 'antd';
+import {connect} from 'dva';
+import {Link} from 'dva/router';
+import {Button, Form, Radio, Input, Tabs, Card, Badge, Icon, Modal, Progress} from 'antd';
 import UnlockByMetaMask from './UnlockByMetaMask'
 import UnlockByKeystore from './UnlockByKeystore'
+import {create} from "Loopring/ethereum/account"
```

Comment:
move this loopring function to model actions

Reply:
slove